.gitignore

```diff
@@ -5,3 +5,5 @@ draytek_exporter
 /.release
 /.tarballs
 /vendor
+/.idea
+/*.iml
```

Comment:
Please don't include your local editor ignores in the repo. Use [`git config --global core.excludesFile`](https://sebastiandedeyne.com/setting-up-a-global-gitignore-file/).